### ISSUE TITLE
Office 365 Security & Compliance	links to this, but apparently do not have correct licence for this

### ISSUE THREAD

**Issue by JamesHoggIOH** [MicrosoftDocs/OfficeDocs-o365seccomp#516]
Your licencing and linking of products is a mess and wastes so much time for users and your team. I have a clear link to  Office 365 Cloud App Security, which says "Your subscription allows you to use Office 365 Cloud App Security!", yet after talking with 3 MS employees all stating that is not their department it was determined that my licence does not have rights to full Cloud App Security, just a limited version. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 88cd56ef-e53f-269a-e741-6057b0ab068b
* Version Independent ID: 6e60eb0b-89e1-1865-77da-a56d1bbcf748
* Content: [Manage OAuth apps using Office 365 Cloud App Security](https://docs.microsoft.com/en-us/office365/securitycompliance/manage-app-permissions-in-ocas)
* Content Source: [SecurityCompliance/manage-app-permissions-in-ocas.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/manage-app-permissions-in-ocas.md)
* Service: **o365-seccomp**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by denisebmsft** [comment]
Hello @JamesHoggIOH, I agree it can be confusing. There's Microsoft Cloud App Security, and Office 365 Cloud App Security. Office 365 Cloud App Security does have fewer features than Microsoft Cloud App Security. The differences are documented here: [https://docs.microsoft.com/en-us/cloud-app-security/editions-cloud-app-security-o365](https://docs.microsoft.com/en-us/cloud-app-security/editions-cloud-app-security-o365). Is this what you are running into, that you were thinking you had Microsoft Cloud App Security and instead you have Office 365 Cloud App Security? I just want to be clear on what the issue is.

**Comment by denisebmsft** [comment]
Hello @JamesHoggIOH. I'm going to close this issue out for now, but if you still have questions, please feel free to reach out.

**Comment by JamesHoggIOH** [comment]
Thanks Denise,

Seems the Microsoft 365 Admin Centers have changed once again with now separate Security and Compliance Admin Sites…. but wait if I then click on certain policies I get to https://protection.office.com<https://protection.office.com/> again which is old Security & Compliance Center with the page I am discussing. I understand the benefit of adding features and simplifying, but the regular half baked changes, with incorrect help documentation, make it almost impossible to administer MS cloud products.

For issue I reported it is clear that even internally you guys don’t understand what you are and aren’t giving access to on certain subscriptions and what your naming of these solutions subsets are to be called.

Issue in more detail:

The page where I start is in MS 365 Admin Center https://protection.office.com/apppermissions, which has link to https://docs.microsoft.com/en-us/office365/securitycompliance/office-365-cas-overview?redirectSourcePath=%252farticle%252f81f0ee9a-9645-45ab-ba56-de9cbccab475
[cid:image001.jpg@01D4E943.4ED3A1C0]

, which has section on OAuth apps, but I do not see my dashboard as show in the help documentation (due to you infer only having “Office 365” subscription type) and most of the links in menu don’t do anything on my [https://iohsolutions.portal.cloudappsecurity.com/#/discovery] site.

Your github page [https://docs.microsoft.com/en-us/office365/securitycompliance/manage-app-permissions-in-ocas] which is titled “Manage OAuth apps using Office 365 Cloud App Security”, has the instructions to access Investigate, which you tell me is not available for “Office 365” subscription type of Cloud App Security.

Basically all just confusing and a waste of administrator’s time trying to figure out MS products. Spent almost a day of time and a lot of MS staff time too, with just the question of wanting to know how I remove OAuth permission for a third party cloud app.


[cid:image002.jpg@01D4E943.4ED3A1C0]

James Hogg
Managing Director
t: 02 9890 3655 | m: 0418 426 567
e: james.hogg@ioh.net | www.ioh.net

For more information please visit
www.yourgoalsyourway.com.au

IOH Disclaimer
​​This message is confidential and should only be used by the intended addressee. If you were sent this email by mistake please inform us by reply email and then destroy the message. The contents of this email are the opinions of the author and do not necessarily represent the views of IOH Injury and Occupational Health
